tests/test_bad_config.rs

```diff
@@ -227,7 +227,10 @@ test!(bad_git_dependency {
         [dependencies]
         foo = { git = "file:.." }
     "#)
-    .file("src/lib.rs", "");
+    .file("src/lib.rs", "").file(".cargo/config", r#"
+    [net]
+    retry=0
```

Comment:
Yes. These are leftover from before the trait. I should be able to remove these.